src/kbmod/search/image_kernels.cu

```diff
@@ -20,7 +20,7 @@ namespace search {
  * Device kernel that convolves the provided image with the psf
  */
 __global__ void convolve_psf(int width, int height, float *source_img, float *result_img, float *psf,
-                            int psf_radius, int psf_dim, float psf_sum) {
+                             int psf_radius, int psf_dim, float psf_sum) {
     // Find bounds of convolution area
     const int x = blockIdx.x * CONV_THREAD_DIM + threadIdx.x;
     const int y = blockIdx.y * CONV_THREAD_DIM + threadIdx.y;
```

Comment:
I'm not sure what you mean here. The change is strictly regarding indexing convention, not the actual memory layout of the arrays. We've always used a row-major ordering because that's the default NumPy ordering. The difference is between `get_pixel(x, y)` and `get_pixel(y, x)`-like signatures of methods. It made sense to me to use the `yx`, i.e. `ij`, notation because I internalized these values as indices, but you made some good points about leaving some functions as `x, y` because it makes sense to put an object on a coordinate - not index. 

I don't think this changes anything about memory layout though....

Reply:
Good point. I think we are all good here.